packages/communication-react/CHANGELOG.stable.md

```diff
@@ -4,76 +4,39 @@ This log was last generated on Mon, 11 Dec 2023 21:24:49 GMT and should not be m
-- Bump GitHub workflows to node v20 ([PR #3856](https://github.com/azure/communication-ui-library/pull/3856) by 3941071+emlynmac@users.noreply.github.com)
-- patch updates to deps ([PR #3847](https://github.com/azure/communication-ui-library/pull/3847) by 94866715+dmceachernmsft@users.noreply.github.com)
-- Add required field to sample textfields ([PR #3869](https://github.com/azure/communication-ui-library/pull/3869) by 2684369+JamesBurnside@users.noreply.github.com)
-- Update Rooms page in storybook ([PR #3832](https://github.com/azure/communication-ui-library/pull/3832) by miguelgamis@microsoft.com)
-- Upgrade rooms client to versions 1.1.0-beta.1 and 1.1.0 ([PR #3819](https://github.com/azure/communication-ui-library/pull/3819) by 79475487+mgamis-msft@users.noreply.github.com)
-- Script to autogenerate api diffs ([PR #3799](https://github.com/azure/communication-ui-library/pull/3799) by 2684369+JamesBurnside@users.noreply.github.com)
-- Update Calling dependency to use the latest version 1.18.1 ([PR #3738](https://github.com/azure/communication-ui-library/pull/3738) by 94866715+dmceachernmsft@users.noreply.github.com)
+- Add additional telemetry to userAgent that indicates either Chat Composite, Call Composite, Call with Chat composite or Stateful components.  ([PR #3481](https://github.com/azure/communication-ui-library/pull/3481, https://github.com/azure/communication-ui-library/pull/3779) by 73612854+palatter@users.noreply.github.com)
+- Update Calling dependency to use the latest version 1.19.1 ([PR #3831](https://github.com/azure/communication-ui-library/pull/3831) by 94866715+dmceachernmsft@users.noreply.github.com)
+- Patch updates to dependencies ([PR #3847](https://github.com/azure/communication-ui-library/pull/3847) by 94866715+dmceachernmsft@users.noreply.github.com)
 - Add console errors for ended call scenarios for Rooms in CallComposite ([PR #3874](https://github.com/azure/communication-ui-library/pull/3874) by 79475487+mgamis-msft@users.noreply.github.com)
-- Update rush/pnpm ([PR #3851](https://github.com/azure/communication-ui-library/pull/3851) by 3941071+emlynmac@users.noreply.github.com)
 - Update communication-chat stable to 1.4.0 ([PR #3866](https://github.com/azure/communication-ui-library/pull/3866) by 107075081+Leah-Xia-Microsoft@users.noreply.github.com)
-- add storybook documentation about custom branding in more places ([PR #3839](https://github.com/azure/communication-ui-library/pull/3839) by 2684369+JamesBurnside@users.noreply.github.com)
-- Update to TypeScript 4.6.3 minimum ([PR #3862](https://github.com/azure/communication-ui-library/pull/3862) by 3941071+emlynmac@users.noreply.github.com)
-- Update Immer to v10 and other  packages. ([PR #3854](https://github.com/azure/communication-ui-library/pull/3854) by 3941071+emlynmac@users.noreply.github.com)
-- Update to rollup 4.7.0 ([PR #3894](https://github.com/azure/communication-ui-library/pull/3894) by 3941071+emlynmac@users.noreply.github.com)
```

Comment:
Same here for the `Update to TypeScript 4.6.3 minimum` one and `Update to rollup 4.7.0`

Reply:
There was a more recent upgrade to TypeScript 5.3.3, on line 25. I think we should only mention the latest.

For rollup, I think we should not include devdependencies updates because it is not relevant for Contoso to know. I could be wrong though. Please let know.